manifest.json

```diff
@@ -18,7 +18,7 @@
 		"email": "maniackc_dev@crudelis.fr"
 	}],
     "requirements": {
-        "yunohost": ">= 3.8.1"
+        "yunohost": ">= 4.0.0"
```

Comment:
I don't see any "helper" that requires ynh >= 4.0, I think we should keep it compatible with stretch, especially for people having problems with php